### PR TITLE
[BASE][DLL][ROSAPPS][SCREENSAVERS] Remove deprecated legacy Win16 resource attributes.

### PR DESCRIPTION
**Pull request by HBelusca**
## Purpose

The resource attributes DISCARDABLE and MOVEABLE are legacy Win16 attributes that are ignored in 32-bit Windows (and ReactOS). Remove them in our code base.
⚠️ Left unchanged in 3rd-party rosapps/lib/vfdlib module.

See: https://learn.microsoft.com/en-us/windows/win32/menurc/common-resource-attributes

## Proposed changes

- Remove DISCARDABLE attribute.
- Remove MOVEABLE attribute.

### CODE REVIEW

File: dll/win32/devmgr/devmgr.rc [hunk 1]
```diff
@@ -18,7 +18,7 @@ IDI_MAIN_ICON   ICON    "resources/computer.ico"
 IDB_ROOT_IMAGE  BITMAP  "resources/root.bmp"
 
 /* main toolbar icons */
-IDB_TOOLBAR    BITMAP DISCARDABLE "resources/toolbar.bmp"
+IDB_TOOLBAR     BITMAP  "resources/toolbar.bmp"
```
Comment (SigmaTel71) [inline dll/win32/devmgr/devmgr.rc:21]:
```suggestion
IDB_TOOLBAR    BITMAP  "resources/toolbar.bmp"
```

**Comment by SergeGautherie** [comment]
> Left unchanged in 3rd-party rosapps/lib/vfdlib module.

It could be done there too, with adding a reminder comment at the beginning of the file.

https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=DISCARDABLE
```
modules/rosapps/lib/vfdlib/vfdlib.rc	
  30 1 TEXTINCLUDE DISCARDABLE 
  35 2 TEXTINCLUDE DISCARDABLE 
  41 3 TEXTINCLUDE DISCARDABLE 
 157 GUIDELINES DESIGNINFO DISCARDABLE 
 194 IDI_VFD_ICON            ICON    DISCARDABLE     "res\\vfd.ico"
 195 IDI_IMAGE_ICON          ICON    DISCARDABLE     "res\\image.ico"
 196 IDI_CONFIG_ICON         ICON    DISCARDABLE     "res\\config.ico"
 198 IDI_VFD_ICON            ICON    DISCARDABLE     "res/vfd.ico"
 199 IDI_IMAGE_ICON          ICON    DISCARDABLE     "res/image.ico"
 200 IDI_CONFIG_ICON         ICON    DISCARDABLE     "res/config.ico"
```

**Comment by whindsaks** [comment]
> It could be done there too

But it would not be worth it if any kind of syncing will be done in the future.

**Comment by SergeGautherie** [comment]
> > It could be done there too
> 
> But it would not be worth it if any kind of syncing will be done in the future.

A "similar" diff was done for `vfdmsg_lib.mc`.
An upstream update seems unlikely, as last version is dated 2008.

**Review by oleg-dubinskiy — APPROVED** [review]

**Comment by oleg-dubinskiy** [comment]
Please fix Clang build, OK otherwise.

**Comment by tkreuzer** [comment]
> A "similar" diff was done for `vfdmsg_lib.mc`. An upstream update seems unlikely, as last version is dated 2008.

And all of thise changes are utterly useless and should have never been done.

**Review by tkreuzer — APPROVED** [review]

**Pull request merged**